docs/developer-networking.md

```diff
@@ -33,9 +33,9 @@ It can be very helpful to add the following lines to your `/etc/hosts` file to a
 ```
 
 ## Certificate
-We'll generate a self-signed certificates for nginx-proxy to use to serve our site over ssl.  Before you begin this, make sure you are in the root directory of the application.
+We'll generate a self-signed certificates for traefix to serve our site over ssl. Before you begin this, make sure you have pulled down the Virtual-Machines repo or have a traefik proxy up and running.
```

Comment:
```suggestion
We'll generate a self-signed certificates for traefik to serve our site over ssl. Before you begin this, make sure you have an installation of the [traefik proxy](http://traefik.io) up and running.
```

Adjusting since only GR employees will have access to the Virtual-Machines repo